apis/routes.go

```diff
@@ -0,0 +1,29 @@
+// Copyright Contributors to the L3AF Project.
+// SPDX-License-Identifier: Apache-2.0
+
+package apis
+
+import (
+	"context"
+	"github.com/l3af-project/l3afd/apis/handlers"
+	"github.com/l3af-project/l3afd/kf"
+	"github.com/l3af-project/l3afd/routes"
+)
+
+func apiRoutes(ctx context.Context, kfcfg *kf.NFConfigs) []routes.Route {
+
+	r := []routes.Route{
+		routes.Route{
+			Method:      "POST",
+			Path:        "/l3af/configs/{version}/update",
+			HandlerFunc: handlers.UpdateConfig(ctx, kfcfg),
+		},
+		routes.Route{
+			Method:      "GET",
+			Path:        "/l3af/configs/{version}/{iface}",
```

Comment:
Can we add a new route that would get the config for all interfaces since the POST is not scoped per interface?  We can keep this one to get just the interface config which is nice to have.